SpecialRequestWiki.php

```diff
@@ -151,3 +151,11 @@ function handleRequestWikiFormInput() {
 		$this->getOutput()->addHTML( '<div class="successbox">' . $this->msg( 'requestwiki-success', $idlink )->plain() . '</div>' );
 	}
 }
+
+function has_hyphens($string)
+{
+    return preg_match('/[\-]/', $string);
+}
+
+if(has_hyphens('-'))
```

Comment:
This '-' should be replaced by the variable that is used to determine the url/DB name. 

That or I don't know what I'm talking about.